tests/test_ordered_serialization.py

```diff
@@ -0,0 +1,58 @@
+# -*- coding: utf-8 -*-
+from schematics.datastructures import OrderedDict
+from schematics.models import Model
+from schematics.types import StringType, IntType
+
+
+def test_serialize_field_orders_one_two_three():
+    class Example(Model):
+        one = StringType()
+        two = IntType()
+        three = StringType()
+
+        class Options:
+            fields_order = ['one', 'two', 'three']
+
+    p1 = Example({'one': 'a', 'two': 2, 'three': '3'})
+    serialized = p1.to_native()
+    comp = OrderedDict()
+    comp['one'] = 'a'
+    comp['two'] = 2
+    comp['three'] = '3'
+    assert serialized == comp
```

Comment:
But isn't the whole point checking if returned value is in right order? I am not sure that I understand what your proposal is... Would you mind giving an example of what would you add in this test using variable names that I used?